src/Tribe/Main.php

```diff
@@ -1718,30 +1738,7 @@ public function localizeAdmin() {
 		 * @return void
 		 */
 		public function printLocalizedAdmin() {
-			$object_name = 'TEC';
-			$vars        = $this->localizeAdmin();
-
-			$data = "var $object_name = {\n";
-			$eol  = '';
-			foreach ( $vars as $var => $val ) {
-
-				if ( gettype( $val ) == 'array' || gettype( $val ) == 'object' ) {
-					$val = json_encode( $val );
-				} else {
-					$val = '"' . esc_js( $val ) . '"';
-				}
-
-				$data .= "$eol\t$var: $val";
-				$eol = ",\n";
-			}
-			$data .= "\n};\n";
-
-			echo "<script type='text/javascript'>\n";
-			echo "/* <![CDATA[ */\n";
-			echo $data;
-			echo "/* ]]> */\n";
-			echo "</script>\n";
-
+			wp_localize_script( 'tribe-events-admin', 'TEC', $this->localizeAdmin() );
```

Comment:
A good question, though luckily one that I checked :D It _is_ being used by `events-community` after `tribe-events-admin` has already been enqueued.